src/dp_abstraction/aca_dataplane_ovs.cpp

```diff
@@ -72,6 +74,35 @@ static bool aca_lookup_subnet_info(GoalState &parsed_struct, const string target
   return false;
 }
 
+static bool aca_lookup_auxgateway_info(GoalState &parsed_struct, const string targeted_vpc_id,
```

Comment:
I understand this is borrowed logic from aca_lookup_subnet_info please update the commands accordingly.